src/screens/AccountNew.js

```diff
@@ -74,7 +74,7 @@ class AccountNewView extends React.Component {
               <AccountIconChooser
                 value={selected && selected.seed && selected.address}
                 onChange={({ address, seed, bip39 }) => {
```

Comment:
Could you please also rename this function here? I forget in which PR we discussed before.